tests/ha/check_after_reboot.pm

```diff
@@ -22,7 +22,9 @@ use utils 'systemctl';
 sub run {
     my $cluster_name = get_cluster_name;
     # In ppc64le and aarch64, workers are slower
-    set_var('TIMEOUT_SCALE', 2) unless (check_var('ARCH', 'x86_64'));
+    my $timeout_scale = get_var('TIMEOUT_SCALE', 2);
```

Comment:
Same here for `scale_timeout`.

Reply:
Cannot use `bmwqemu::scale_timeout` here as this block is only used to update the value of `TIMEOUT_SCALE` in slower workers. All other uses of `TIMEOUT_SCALE` were changed though.